script/bridges/layerZero/IexecLayerZeroBridge.s.sol

```diff
@@ -67,6 +73,20 @@ contract Configure is Script {
         sourceBridge.setPeer(
             targetParams.lzChainId, bytes32(uint256(uint160(targetParams.iexecLayerZeroBridgeAddress)))
         );
+        if (sourceParams.approvalRequired) {
+            RLCLiquidityUnifier rlcLiquidityUnifier = RLCLiquidityUnifier(sourceParams.rlcLiquidityUnifierAddress);
+            bytes32 bridgeTokenRoleId = rlcLiquidityUnifier.TOKEN_BRIDGE_ROLE();
+            rlcLiquidityUnifier.grantRole(bridgeTokenRoleId, address(sourceBridge));
+        } else {
+            RLCCrosschainToken rlcCrosschainToken = RLCCrosschainToken(sourceParams.rlcCrosschainTokenAddress);
+            bytes32 bridgeTokenRoleId = rlcCrosschainToken.TOKEN_BRIDGE_ROLE();
+            rlcCrosschainToken.grantRole(bridgeTokenRoleId, address(sourceBridge));
+        }
+
+        EnforcedOptionParam[] memory enforcedOptions = new EnforcedOptionParam[](1);
+        bytes memory _extraOptions = OptionsBuilder.newOptions().addExecutorLzReceiveOption(70_000, 0); // 70_000 gas limit for the receiving executor and 0 for the executor's value
+        enforcedOptions[0] = EnforcedOptionParam(targetParams.lzChainId, 2, _extraOptions);
+        sourceBridge.setEnforcedOptions(enforcedOptions);
```

Comment:
Let's put this before bridge authorization to have all LZ config then our custom config.